src/DynamoCoreWpf/UI/Prompts/CrashPrompt.xaml.cs

```diff
@@ -42,6 +47,12 @@ public CrashPrompt(CrashPromptArgs args, DynamoViewModel dynamoViewModel)
         {
             InitializeComponent();
 
+            var packageLoader = dynamoViewModel.Model.GetPackageManagerExtension().PackageLoader;
```

Comment:
be careful here - `GetPackageManagerExtension` might return null in certain circumstances - you can do
` dynamoViewModel.Model.GetPackageManagerExtension()?.PackageLoader` - and then proceed if packageLoader is not null.

Reply:
good catch, will fix